src/redis_version.c

```diff
@@ -0,0 +1,42 @@
+#include "redis_version.h"
+#include "redismodule.h"
+#include <assert.h>
+#include <string.h>
+
+int redisMajorVersion;
```

Comment:
Initialize these to 0.

This might also be a good place to add an 'enterprise' variable or whatnot.

The init function should be called from module-init.c; not OnLoad (Which might never be called if we're called as an external library).

Reply:
@mnunberg if we are a library I do not want to call this